doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,286 @@
+.. index:: fix ave/moments
+
+fix ave/moments command
+=======================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   fix ID group-ID ave/moments Nevery Nrepeat Nfreq value1 value2 ... moment1 moment2 ... keyword args ...
+
+* ID, group-ID are documented in :doc:`fix <fix>` command
+* ave/moments = style name of this fix command
+* Nevery = use input values every this many time steps
+* Nrepeat = # of times to use input values for calculating averages
+* Nfreq = calculate averages every this many time steps
+* one or more input variables can be listed
+* value = v_name
+
+  .. parsed-literal::
+
+       c_ID = global scalar calculated by a compute with ID
+       c_ID[I] = Ith component of global vector calculated by a compute with ID, I can include wildcard (see below)
+       f_ID = global scalar calculated by a fix with ID
+       f_ID[I] = Ith component of global vector calculated by a fix with ID, I can include wildcard (see below)
+       v_name = value calculated by an equal-style variable with name
+       v_name[I] = value calculated by a vector-style variable with name, I can include wildcard (see below)
+
+* one or more moments to compute can be listed
+* moment = *mean* or *stddev* or *variance* or *skew* or *kurtosis*, see exact definitions below.
+* zero or more keyword/arg pairs may be appended
+* keyword = *start* or *delay*
+
+  .. parsed-literal::
+
+       *start* args = Nstart
+         Nstart = invoke first after this time step
+       *history* args = Ndelay
```

Comment:
I don't think Ndelay is a good descriptor for history.  I suggest Nrecent instead.
I.e. the history option stores the Nrecent most recent outputs in an array for access by the input script